dart/dynamics/BodyNode.h

```diff
@@ -1035,7 +1035,7 @@ class BodyNode :
   UniqueProperties mBodyP;
 
   /// Whether the node is currently in collision with another node.
-  DEPRECATED(6.0)
+  /// \deprecated DEPRECATED(6.0) See #670 for more detail.
```

Comment:
Removed the used of `DEPRECATED(6.0)` for member variable because there is no way to suppress warnings for this. Instead, I put `DEPRECATED(6.0)` into comment so that we don't miss this when searching deprecated API using this keyword.